public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -126,35 +126,48 @@ export function DashList(props: PanelProps<DashListOptions>) {
   ];
 
   const css = useStyles2(getStyles);
+
+  const renderList = (dashboards: Dashboard[]) => (
+    <ul>
+      {dashboards.map((dash) => (
+        <li className={css.dashlistItem} key={`dash-${dash.id}`}>
+          <div className={css.dashlistLink}>
+            <div className={css.dashlistLinkBody}>
+              <a className={css.dashlistTitle} href={dash.url}>
+                {dash.title}
+              </a>
+              {dash.folderTitle && <div className={css.dashlistFolder}>{dash.folderTitle}</div>}
+            </div>
+            <IconToggle
+              aria-label={`Star dashboard "${dash.title}".`}
+              className={css.dashlistStar}
+              enabled={{ name: 'favorite', type: 'mono' }}
+              disabled={{ name: 'star', type: 'default' }}
+              checked={dash.isStarred}
+              onClick={(e) => toggleDashboardStar(e, dash)}
+            />
+          </div>
+        </li>
+      ))}
+    </ul>
+  );
+
+  const renderPreviews = (dashboards: Dashboard[]) => (
+    <div>
+      TODO!!! PREVIEWS!!!!
```

Comment:
@ashharrison90 -- I *think* the `FixedSizeGrid` goes in here, but we don't really need a virtualized view since the count is limited anyway.